mungegithub/mungeopts/mungeopts.go

```diff
@@ -46,7 +49,7 @@ var (
 	}
 )
 
-func RegisterOptions(opts *options.Options) {
+func RegisterOptions(opts *options.Options) sets.String {
```

Comment:
`RegisterOptions registers options used by mungers that serve web services and returns any options that should require a restart when they are changed.`

take it or leave it, LGTM